test/cli/test-inline-suppress.py

```diff
@@ -31,3 +32,19 @@ def test_backwards_compatibility():
 
     ret, stdout, stderr = cppcheck(['--inline-suppr', 'proj-inline-suppress/3.cpp'])
     assert stderr == ''
+
+def test_unused_function_suppression():
+    # Create compile_commands.json
+    compile_commands = os.path.join('proj-inline-suppress-unusedFunction', 'compile_commands.json')
+    prjpath = os.path.join(os.getcwd(), 'proj-inline-suppress-unusedFunction')
+    j = [{'directory': prjpath,
+          'command': 'g++ -I"' + prjpath + '" -std=gnu++1z -o ' + os.path.join(prjpath, 'B.cpp.o') + ' -c ' + os.path.join(prjpath, 'B.cpp'),
+          'file': os.path.join(prjpath, 'B.cpp')},
+         {'directory': prjpath,
+          'command': 'g++ -I"' + prjpath + '" -std=gnu++1z -o ' + os.path.join(prjpath, 'A.cpp.o') + ' -c ' + os.path.join(prjpath, 'A.cpp'),
+          'file': os.path.join(prjpath, 'A.cpp')}]
+    with open(compile_commands, 'wt') as f:
+        f.write(json.dumps(j, indent=4))
+    # Run cppcheck
+    ret, stdout, stderr = cppcheck(['--inline-suppr', '--enable=information', '--error-exitcode=1', '--project=./proj-inline-suppress-unusedFunction/compile_commands.json'])
```

Comment:
I think that `--enable=information` should be changed.

Reply:
I changed it to --enable=all. Not sure if that is what you intended, but it works now!